ios/HyperSdkReact.mm

```diff
@@ -154,6 +282,12 @@ -(void)stopObserving {
     }
 }
 
+RCT_EXPORT_METHOD(updateMerchantViewHeight: (NSString * _Nonnull) tag height: (NSNumber * _Nonnull) h) {
```

Comment:
@Georgepadannamackal When do they need to call this function? And will merchant be able to get the exact height of the view?